mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/17 - network diagram

Closes #17 

To test:
- Run different scenarios and open Network View
- Make sure all computers on same network have lines connecting them
- Toggle connections on and off and make sure lines update properly
- Make sure new computers/devices that are bought show up in Network View
- Make sure network toggles are disabled for static computers/devices (the ones that in the original game say "you can't modify this")
- Make sure all network lines are visible (even if you have to scroll the view a lot)
